controller/placeholder.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const _ = require('lodash');
+
+const logger = require('pelias-logger').get('api');
+const logging = require( '../helper/logging' );
+const Document = require('pelias-model').Document;
+
+function synthesizeDocs(result) {
+  return result.lineage.map((hierarchy) => {
```

Comment:
it's possible that some WOF records have no hierarchy and so might have no lineage (as in not even an empty array):

- https://github.com/pelias/placeholder/blob/master/prototype/wof.js#L23
- https://github.com/whosonfirst-data/whosonfirst-data/issues/739